kotlin-csstype/gradle.properties

```diff
@@ -0,0 +1 @@
+description=Kotlin wrapper for CSSType
```

Comment:
An author field must be present for publishing to work.

Reply:
fixed